tests/data/state_estimation/single-line-load/input.json

```diff
@@ -62,7 +62,15 @@
       "measured_terminal_type": 4,
       "p_measured": 2e6,
       "q_measured": 0.0,
-      "power_sigma": 1e20
+      "power_sigma": 1e3
```

Comment:
What was the reason to modify the existing case too?
Was there an interaction with high sigma 1e20 and numerical limits sigma?

Reply:
as discussed yesterday: the original test didn't make sense - especially not for a test named `single-line-load`. we want to actually test the behaviour in edge cases (e.g. when two measurements are not in agreement, then we still have to handle it correctly - especially because "not in agreement" is also a statistically indetermined question without specifying confidence intervals which we don't have)